_posts/2021-12-14-dec-community-call-summary.md

```diff
@@ -0,0 +1,110 @@
+Then David introduced two lightning talks
+that would help guide community discussion in the breakout rooms.
+
+The [first talk](https://youtu.be/0vdfNojvlck?t=645)
+from [Jeff Carver](http://carver.cs.ua.edu/)
+provided a crash course in constructive code reviews,
+answering a couple of key questions:
+* What are the some of the goals of doing code review?
+* How should I approach the review if I'm the one submitting code,
+  or if I'm reviewing someone else's code?
+
+In the [second talk](https://youtu.be/0vdfNojvlck),
+[Pat Schloss ](http://www.schlosslab.org/)
+introduced the idea of
+[Code Clubs](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008119#sec009),
+which he developed in his research group
+to achieve many of the same goals of code review,
+such as knowledge sharing.
+
+Interestingly, both Jeff and Pat made points
+about how reading code, reviewing others' code,
+and getting feedback on how we write code
+can fill some of the gaps
+left in traditional computer science education.
+The traditional approach
+often encourages students to learn by writing, not reading.
+David pointed out that Greg Wilson
+had shared, in the US-RSE Slack, the book
+["The Programmer's Brain"](https://www.manning.com/books/the-programmers-brain)
+that makes similar observations about how
+coding is taught,
+and describes methods for improving reading comprehension.
+
+## Breakout room discussion
+
+Then the attendees entered the breakout rooms to discuss.
+Each group took notes, and then
+came back to the main room at the end of the call
+[to report out](https://youtu.be/0vdfNojvlck?t=1575).
+One common idea across groups was
+that of automating as much as possible---with
+linters and continuous integration, for example---so that
+high-bandwidth face-to-face code review
+could focus on big-picture planning.
+At the same time, many RSEs were working
+in situations where this one-size-fits-all approach
+did not lend itself to review: e.g.,
+a group with two main developers, where
+undergraduate CS students make
+limited contributions that require very specific,
+tailored recommendations such as "Add a comment here".
+
+## Future community calls
+
+The range of responses from the breakout rooms
+suggest that code review will continue to be a topic
+of interest for US-RSE and the global RSE community.
+One aspect of code review that was not addressed
+in this community call is whether
+good practices as they are currently understood
+can realistically be put into practice by all RSEs.
+A "lone wolf" developer or a
+single scientist writing analysis code in a small lab  
+may not have access to a local community of
+software engineers that they can turn to for review.
+Could US-RSE help address this
+by providing some sort of virtual community?
+These accessibility issues could be a topic for a future call.
+Please feel free to suggest other topics
+that may be of interest to you on the repository we've set up:  
```

Comment:
ok, the line break here is due to the two spaces after the ":", which are a markdown code for a linebreak - please remove them